vision/beta/cloud-client/src/main/java/com/example/vision/AsyncBatchAnnotateImagesGcs.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.vision;
+
+// [START vision_detect_async_batch_annotate_images_beta]
+
+import com.google.api.core.ApiFuture;
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.Storage.BlobListOption;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.vision.v1p4beta1.AnnotateImageRequest;
+import com.google.cloud.vision.v1p4beta1.AsyncBatchAnnotateImagesRequest;
+import com.google.cloud.vision.v1p4beta1.BatchAnnotateImagesResponse.Builder;
+import com.google.cloud.vision.v1p4beta1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1p4beta1.Feature;
+import com.google.cloud.vision.v1p4beta1.Feature.Type;
+import com.google.cloud.vision.v1p4beta1.GcsDestination;
+import com.google.cloud.vision.v1p4beta1.Image;
+import com.google.cloud.vision.v1p4beta1.ImageAnnotatorClient;
+import com.google.cloud.vision.v1p4beta1.ImageSource;
+import com.google.cloud.vision.v1p4beta1.OutputConfig;
+
+import com.google.longrunning.Operation;
+import com.google.protobuf.util.JsonFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class AsyncBatchAnnotateImagesGcs {
+
+  /**
+   * Performs asynchronous batch annotation of images on Google Cloud Storage
+   * @param gcsSourcePath The path to the remote file on Google Cloud Storage
+   * @param gcsDestinationPath The path to the annotated image file
+   * @throws Exception on errors
+   */
+
+  public static void asyncBatchAnnotateImagesGcs(String gcsSourcePath, String gcsDestinationPath)
```

Comment:
In the new style we're adding commented out variables to demonstrate what the parameters should be and simplifying the method comment. 

```
// Performs asynchronous batch annotation of images on Google Cloud Storage
public static void asyncBatchAnnotateImagesGcs(String gcsSourcePath, String gcsDestinationPath) throws Exception {
    // String gcsSourcePath = "gs://YOUR_BUCKET_ID/path_to_you_data";
    // String gcsDestinationPath = "gs://YOUR_BUCKET_ID/path_to_store_annotation";
```

Reply:
Fixed across all files in latest commit.